### ISSUE TITLE
[FEATURE] Improve the Admin API

### ISSUE THREAD

**Issue by mingyifei** [streamnative/aop#863]

**Comment by KamenRiderKuuga** [comment]
According to my understanding, these Admin APIs are all HTTP APIs. Even without these HTTP APIs, we can still perform similar following operations through the `AMQP` protocol, right?

1. Get the queue list
2. Create a queue
3. Query the details of a queue
4. Delete a queue

**Comment by mingyifei** [comment]
> 

Yes, but we need to be consistent with the original Rabbitmq management backend for perfect cluster migration

**Comment by mingyifei** [comment]
> According to my understanding, these Admin APIs are all HTTP APIs. Even without these HTTP APIs, we can still perform similar following operations through the `AMQP` protocol, right?
> 
> 1. Get the queue list
> 2. Create a queue
> 3. Query the details of a queue
> 4. Delete a queue


Yes, but we need to be consistent with the original Rabbitmq management backend for perfect cluster migration
